psutil/tests/__init__.py

```diff
@@ -29,6 +29,7 @@
 import threading
 import time
 import traceback
+import unittest
```

Comment:
Instead of doing this can you please change all other test files and `import unittest` directly instead of `from psutil.test import unittest`?